client/js/components/procedure/SegmentsList/SegmentsList.vue

```diff
@@ -664,6 +694,147 @@ export default {
       this.$refs.customSearch.reset()
     },
 
+    /**
+     *
```

Comment:
Nonblocking: While variable naming and the JSDoc block surely helps a lot in getting a grasp on this rather complex method, i would love to see a high level description on what it does. I could not exactly tell what it does after scanning through, tbh... i guess it takes the current filter selection (as request params), calls the rpc route with it, and updates filter state with the results... ? :)